views/index.handlebars

```diff
@@ -5,7 +5,7 @@
             <div class="col  m6">
                 <div class="card">
                     <div class="card-image" style="background-color: #EFE5A1;">
-                        <img src="./assets/images/logo2.png" style="height:75px; width:300px; " />
+                        <img src="/assets/images/logo2.png" style="height:75px; width:300px; " />
```

Comment:
Just an example -- you don't need to use relative paths if express is serving the public folder.